contents/docs/experiments/statistics-bayesian.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Bayesian statistics
+---
+
+PostHog's Bayesian analysis engine provides a statistically rigorous yet user-friendly approach to experimentation. Instead of p-values and confidence intervals, it calculates the **probability that one variant is better than another** and directly answers the question: "Should I ship this change?"
+
+For example, instead of saying "statistically significant at p < 0.05," you get intuitive statements like "There's a 96% chance that variant B increases conversion rate."
+
+The engine transforms raw experiment data through a clear 5-step pipeline:
+
+- **Aggregate** user-level data into sufficient statistics
+- **Validate** data quality and sample size requirements
+- **Calculate** effect size and its uncertainty
+- **Update** beliefs using Bayesian inference
+- **Generate** intuitive probability-based results
+
+## Input data: What goes into the analysis
+
+The Bayesian engine processes three types of metrics:
+
+### Funnel metrics
+
+Track whether users complete a specific action like "Did they complete checkout?" These are yes/no outcomes that we analyze as conversion rates. ([Learn more about funnel metrics](/docs/experiments/metrics#funnel))
+
+### Mean metrics
+
+Measure numeric values for each user, such as revenue per user or session duration. We calculate the average value across all users in each variant. ([Learn more about mean metrics](/docs/experiments/metrics#mean))
```

Comment:
```suggestion
Measure numeric values for each user, such as revenue per user or session duration. We calculate the average value across all users in each variant. [Learn more about mean metrics](/docs/experiments/metrics#mean).
```

We can even use OSButtons on the new website for proper links to learn more.

They can be of the secondary variant so they're not super high weight, see:

https://github.com/PostHog/posthog.com/pull/13299/files#diff-73a4a65b5b0c5d3f48888392562624b4c4e447f6355650a0dec5d76ffe6b403cR113

Up to you with what you prefer